src/utils.js

```diff
@@ -1,5 +1,7 @@
 import parse from "url-parse";
 
+export const defaultPortalUrl = "https://siasky.net";
```

Comment:
in my opinion we shouldn't provide default here, it should be explicitly defined by consumer

Reply:
Why do you say that? In my understanding we want the SDKs to be as quick and easy to use as possible. In the other SDKs we provide a default portal URL which allows someone to get started with the library as easily as calling `UploadFile()`. And after this PR somoene could just call `SkynetClient()` and get started making calls.

Reply:
It is easy to obscure the otherwise explicit definition of the portal if we make our portal a default. This could cause bad user experience if people start relying on https://siasky.net free service as a de facto way to use the library and our portal starts having problems or we take it down for updates or sth. We should include it in the examples but the actual library should stay portal agnostic.

@m-cat FLW: the same should apply to all our SDKs